app/controllers/admins/panel_controller.rb

```diff
@@ -10,6 +10,22 @@ def index
     @drivers = Driver.order('active DESC').all
   end
 
+  def edit_driver_photo
```

Comment:
I think it's better to name it with `update`, not `edit`, as far as you're update something in the database during this action.